### PR TITLE
[TACHYON-373] Add tachyon-perf-repeat script

### PR DESCRIPTION
**Pull request by rvesse**
Adds a new script that allows running a performance test some number of times which also collates and summarises the results for you

### CODE REVIEW

File: perf/bin/tachyon-perf-repeat [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+bin=`cd "$( dirname "$0" )"; pwd`
+
+DEFAULT_PERF_LIBEXEC_DIR="$bin"/../libexec
+TACHYON_PERF_LIBEXEC_DIR=${TACHYON_PERF_LIBEXEC_DIR:-$DEFAULT_PERF_LIBEXEC_DIR}
+. $TACHYON_PERF_LIBEXEC_DIR/tachyon-perf-config.sh
+
+# Validate test case name
+case ${TEST} in
+SimpleRead|SimpleWrite|Metadata|SkipRead|Workflow)
```
Comment (calvinjia) [inline perf/bin/tachyon-perf-repeat]: Is workflow in the perf branch? Also, should Mixture be here?

**Comment by calvinjia** [comment]
I noticed there are a few hardcoded actions (ie. generate data, clean workspace after run), and I think it might be better to have those as flags. This way the scripts do not need to be updated each time new benchmarks are added, though the command to run it will be more complex. An alternative is we have a mechanism for tests to describe their properties to the scripts (unrelated to this PR).

File: perf/bin/tachyon-perf-repeat (hunk 1 above)
Comment (calvinjia) [inline perf/bin/tachyon-perf-repeat]: Oh also maybe this is not necessary. In tachyon-perf.sh, it fails at run time (but its uglier). The benefit being the cases do not need to be hardcoded.

**Comment by calvinjia** [comment]
Also, could you please open a JIRA ticket for this?

**Comment by rvesse** [comment]
@calvinjia I have rewritten the script so all the test case specific behaviours are now command line options so that this script is flexible

Also filed as https://tachyon.atlassian.net/browse/TACHYON-373

**Comment by calvinjia** [comment]
+1 LGTM

**Pull request merged**